lib/OrdinaryDiffEqPRK/src/algorithms.jl

```diff
@@ -1,10 +1,10 @@
-"""
-KuttaPRK2p5: Parallel Explicit Runge-Kutta Method
-A 5 parallel, 2 processor explicit Runge-Kutta method of 5th order.
-
-These methods utilize multithreading on the f calls to parallelize the problem.
-This requires that simultaneous calls to f are thread-safe.
-"""
+@doc generic_solver_docstring("A 5 parallel, 2 processor method of 5th order.
+        ",
+        "KuttaPRK2p5",
+        "Explicit Runge-Kutta Method",
+        "REFERENCE TBD",
+        "- `threading`: TBD",
+        "threading = true,")
```

Comment:
@ChrisRackauckas reference and description of threading needed.
I'm confused what the difference is between `threading` here and `thread` in e.g. Tsit85.

Reply:
It's the same. It's an oversight that these were named differently and we should correct this with the v7

Reply:
@ChrisRackauckas you did not give a reference.

Reply:
IIRC it's https://www.jstor.org/stable/2158288